src/Swashbuckle.AspNetCore.SwaggerGen/SchemaGenerator/MemberInfoExtensions.cs

```diff
@@ -2,22 +2,22 @@
 using System.Linq;
 using System.Reflection;
 using Microsoft.AspNetCore.Mvc;
-
+using System;
 namespace Swashbuckle.AspNetCore.SwaggerGen
 {
     public static class MemberInfoExtensions
     {
         public static IEnumerable<object> GetInlineOrMetadataTypeAttributes(this MemberInfo memberInfo)
         {
-            var metadataTypeAttribute = memberInfo.DeclaringType.GetCustomAttributes(true)
+            var metadataTypeAttribute = memberInfo?.DeclaringType.GetCustomAttributes(true)
```

Comment:
Like I said, I don't want to null check the extension instance inside the method itself. Rather, I was suggesting the following:

```csharp
var metadataTypeAttribute = memberInfo.DeclaringType?.GetCustomAttributes(true)
```